src/components.test/Quote.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import Quote from '../components/Quote';
+
+describe('Check if it', () => {
+  it('renders Quote component', () => {
+    render(<Quote />);
+  });
+});
+
```

Comment:
- [ ] Great work on implementing the snapshots :clap: but it would be nice if you make tests on user interactions with your application such as clicking a button and then seeing the output as suggested by the previous reviewer :+1:

Hint: You can do this using import `import {fireEvent} from '@testing-library/react'`. You can see this [documentation](https://testing-library.com/docs/dom-testing-library/api-events/) of fireEvent to get an understanding of why and how to use it :+1:

Reply:
@HAMMAS-SALEEM I added that one